### DISCUSSION TITLE
Version 1.24.1

### DISCUSSION THREAD

**Discussion by jangxx** [jangxx/netflix-1080p#24] [General]
- Changed the URL match pattern to hopefully match all of Netflix's new CDN now
- Added log message to indicate that the patched player has been loaded

<hr /><em>This discussion was created from the release <a href='https://github.com/jangxx/netflix-1080p/releases/tag/v1.24.1'>Version 1.24.1</a>.</em>

**Comment by akbargs** [comment]
![image](https://user-images.githubusercontent.com/74965658/119733298-837ebd00-bea3-11eb-9e69-7a4afff1e277.png)

Hi, How to solve this? It can't be turned on at all

**Reply by jangxx** [reply]
Googling around a bit, it seems like this is a restriction that affects the Windows and macOS versions of Chrome (but apparently not every user) and there are only a few workarounds:

- Load the extension as unpacked
- (Windows) Use a group policy to whitelist the extension ID "mkimjnfbbelfbcoecfnpdffkongeffjl": https://support.google.com/chrome/forum/AAAAP1KN0B09NlMAr6uEVc/?hl=en&msgid=ambkrcKpi1cJ&gpf=d/msg/chrome/9NlMAr6uEVc/ambkrcKpi1cJ and then install the extension normally

But you might ask yourself the question why you would even use Chrome on Windows at all to watch Netflix when you could use both the official Netflix app as well as Chromium Edge, which both support higher quality video and audio out of the box. Same for macOS and Safari.
